docs/pages/web/utilities/defaultlabelprovider.js

```diff
@@ -9,6 +10,287 @@ import SandpackExample from '../../../docs-components/SandpackExample.js';
 import noTranslations from '../../../examples/defaultlabelprovider/no-translations.js';
 import translations from '../../../examples/defaultlabelprovider/translations.js';
 
+// Function to generate the table below - need access to fallbackLabels to actually do so
+function getLabelsTable(/* fallbackLabels: { [string]: { [string]: mixed } } */) {
+  return [
+    {
+      'component': 'ActivationCard',
+      'prop': 'accessibilityDismissButtonLabel',
+      'label': 'Dismiss',
+    },
+    {
+      'component': 'Callout',
+      'prop': 'accessibilityDismissButtonLabel',
+      'label': 'Dismiss',
+    },
+    {
+      'component': 'Callout',
+      'prop': 'iconAccessibilityLabelError',
+      'label': 'Error',
+    },
+    {
+      'component': 'Callout',
+      'prop': 'iconAccessibilityLabelInfo',
+      'label': 'Info',
+    },
+    {
+      'component': 'Callout',
+      'prop': 'iconAccessibilityLabelRecommendation',
+      'label': 'Recommendation',
+    },
+    {
+      'component': 'Callout',
+      'prop': 'iconAccessibilityLabelWarning',
+      'label': 'Warning',
+    },
+    {
+      'component': 'Callout',
+      'prop': 'iconAccessibilityLabelSuccess',
+      'label': 'Success',
+    },
+    {
+      'component': 'ComboBox',
+      'prop': 'noResultText',
+      'label': 'No results',
+    },
+    {
+      'component': 'ComboBox',
+      'prop': 'accessibilityClearButtonLabel',
+      'label': 'Clear input',
+    },
+    {
+      'component': 'Link',
+      'prop': 'accessibilityNewTabLabel',
+      'label': 'Opens a new tab',
+    },
+    {
+      'component': 'Modal',
+      'prop': 'accessibilityDismissButtonLabel',
+      'label': 'Dismiss modal',
+    },
+    {
+      'component': 'Popover',
+      'prop': 'accessibilityDismissButtonLabel',
+      'label': 'Dismiss popover',
+    },
+    {
+      'component': 'Pog',
+      'prop': 'iconAccessibilityLabelError',
+      'label': 'Error',
+    },
+    {
+      'component': 'Pog',
+      'prop': 'iconAccessibilityLabelInfo',
+      'label': 'Info',
```

Comment:
Wait, why are we adding default to Pog?

Reply:
DONE (Removed like the previous comment)